src/AdminBar.php

```diff
@@ -32,7 +32,7 @@ public function render( $wp_admin_bar ) {
 
 		$wp_admin_bar->add_node( array(
 			'id'    => 'spinupwp',
-			'title' => 'SpinupWP',
+			'title' => (getenv('SPINUPWP_SITE')) ? 'Cache' : 'SpinupWP',
```

Comment:
@bcmcq we should use `apply_filters` here so that devs can update the menu text if required.

https://developer.wordpress.org/reference/functions/apply_filters/

@bradt if we make this filterable, do you still want to update the menu label to "Cache" or keep it as "SpinupWP"?

Reply:
I think "Cache" is actually better for UX